docs/execution_algorithms/ltfb.rst

```diff
@@ -121,17 +121,322 @@ Python Front-end API Documentation
 The following is the full documentation of the Python Front-end
 classes that are used to implement this execution algorithm.
 
-.. autoclass:: lbann.LTFB
-   :members:
-   :undoc-members:
-   :special-members: __init__
+.. _LTFB:
 
-.. autoclass:: lbann.MetaLearningStrategy
-   :members:
-   :undoc-members:
-   :special-members: __init__
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+lbann.LTFB module
```

Comment:
I don't think "module" is the right word here, as that's meaningful in Python contexts, and that's not how we mean it here. Maybe just "interface"?